### DISCUSSION TITLE
Got some spare PCBs - Can I sell them here?

### DISCUSSION THREAD

**Discussion by qbinary** [sanni/cartreader#444] [General Chat]
I ordered the PCBs for Version 5 and got some spare.
Is it ok if I offer them here @sanni? I live in Germany and would sell them for cheap just to get rid of them.

**Comment by sanni** [comment]
This Github is for development only. I try to keep anything related to money out of it. You can however try the Arduino forum thread here: https://forum.arduino.cc/t/rom-reader-for-super-nintendo-super-famicom-game-cartridges/155260/1551
